app/services/responsibility_service.rb

```diff
@@ -182,24 +182,21 @@ def self.open_prison_nps_offender?(offender)
 
   class CrcRules
     def self.responsibility(offender)
-      if release_date_gt_12_weeks?(offender)
+      # CRC can look at HDC date, NPS is not supposed to
+      earliest_release_date =
+        offender.home_detention_curfew_actual_date.presence ||
+          [offender.automatic_release_date,
+           offender.conditional_release_date,
+           offender.home_detention_curfew_eligibility_date].compact.min
+
+      return nil if earliest_release_date.nil?
+
+      if earliest_release_date > DateTime.now.utc.to_date + 12.weeks
```

Comment:
Can we split this refactor into its own commit? As discussed, I don't think it's related to the primary change in this commit – i.e. displaying N/A for handover dates where the COM is already responsible